ExecuteTests.ahk

```diff
@@ -0,0 +1,50 @@
+﻿#NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
+; #Warn  ; Enable warnings to assist with detecting common errors.
+SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
+SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.
+
+
+
+#Include %A_ScriptDir%\tests\Automated Unit Tests\MiscHelpersTests.ahk
+#Include %A_ScriptDir%\tests\Automated Unit Tests\OCRActionSequencerMainTests.ahk
+;Include %A_ScriptDir%\tests\Automated Unit Tests\SampleTest1.ahk ; don't use this one unless testing the tests, it has an intentional fail
+
+
+
+; We need this class for one main reason: in AHK you can't be dynamic with your #Include statement.
+; We're using %Include% with %A_ScriptDir% in the various files so they can use each other.
+; The thing about %A_ScriptDir% is though, is that it contains the directory of the parent-most
+; caller of the scripts. So in the case of executing our main script, using %A_ScriptDir there returns its
+; directory. This is good, because the scripts it needs to include are also in that directory.
+; But once we're in one of the unit tests, located in a subfolder from where the main script it,
+; %A_ScriptDir% will contain the subfolder where the test resides, meaning that when the main script
+; tries using it to include the scripts that are truly right next to it, the #Include statement
+; tries to look for files in the test's folder.
+;
+; This means we have two options: Make it so %A_ScriptDir% is the same as the main script's (the option
+; that this file's existence give us), or try to %Include% not based on %A_ScriptDir%.
+; The problem with the latter option is that #Include won't evaluate any user variables, only built-in ones
+; (like %A_ScriptDir%). To make things harder, #Include is the first thing evaluated in a script. So you can't
+; even get cute by manipulating built-in variables beforehand (if there are even manipulatable ones, didn't
+; dive too deep here). The one caveat to this is if you put your
```

Comment:
Definitely had a stroke writing this one. Let's try "The one caveat to this is you can put your #Include within a function"